src/main/java/edu/harvard/iq/dataverse/util/json/JsonParser.java

```diff
@@ -211,9 +229,17 @@ public DatasetVersion parseDatasetVersion(JsonObject obj, DatasetVersion dsv) th
             terms.setContactForAccess(obj.getString("contactForAccess", null));
             terms.setSizeOfCollection(obj.getString("sizeOfCollection", null));
             terms.setStudyCompletion(obj.getString("studyCompletion", null));
-            /* License???*/
+
+            // License: only CC0 or NONE is currently supported
+            TermsOfUseAndAccess.License license = TermsOfUseAndAccess.License.NONE;
```

Comment:
@sekmiller and I talked about this the other day and we think by default the license should be `CC0` to match what the GUI does. (Yes, this is a change in behavior, which is what #1991 is about... how through the native API you end up with a license of "null" which is never what we want.) Best would probably be to reference the default license via the existing static variable like this:

    TermsOfUseAndAccess.License license = TermsOfUseAndAccess.defaultLicense;

Obviously, the logic below would need to change to change `CC0` to `NONE` when you see fit (when you supply "NONE" in the JSON).